lib/JMAP/Tester.pm

```diff
@@ -111,46 +125,13 @@ for my $type (qw(api authentication download upload)) {
 }
 
 has ua => (
-  is   => 'ro',
-  lazy => 1,
+  is => 'ro',
   default => sub {
-    my ($self) = @_;
-
-    require LWP::UserAgent;
-    my $ua = LWP::UserAgent->new;
-    $ua->cookie_jar({});
-
-    if ($ENV{IGNORE_INVALID_CERT}) {
-      $ua->ssl_opts(SSL_verify_mode => 0, verify_hostname => 0);
-    }
-
-    return $ua;
+    require JMAP::Tester::UA::LWP;
+    JMAP::Tester::UA::LWP->new;
   },
 );
 
-sub _set_cookie {
```

Comment:
Removing this will break some downstream code I believe (like Ix, etc)

Reply:
I wrote a branch in Ix, rjbs/future, that fixes this, already. :)